shared/chat/conversation/header-area/normal/container.tsx

```diff
@@ -41,31 +41,32 @@ const mapDispatchToProps = (
   onToggleThreadSearch: () => dispatch(Chat2Gen.createToggleThreadSearch({conversationIDKey})),
 })
 
-const mergeProps = (stateProps, dispatchProps) => ({
-  badgeNumber: stateProps._badgeMap.reduce(
-    (res, currentValue, currentConvID) =>
-      // only show sum of badges that aren't for the current conversation
-      currentConvID !== stateProps._conversationIDKey ? res + currentValue : res,
-    0
-  ),
-  channelName: stateProps.channelName,
-  infoPanelOpen: stateProps.infoPanelOpen,
-  muted: stateProps.muted,
-  onBack: dispatchProps.onBack,
-  onOpenFolder: dispatchProps._onOpenFolder,
-  onShowProfile: dispatchProps.onShowProfile,
-  onToggleInfoPanel: dispatchProps.onToggleInfoPanel,
-  onToggleThreadSearch: dispatchProps.onToggleThreadSearch,
-  participants: stateProps._participants.toArray(),
-  pendingWaiting: stateProps.pendingWaiting,
-  smallTeam: stateProps.smallTeam,
-  teamName: stateProps.teamName,
-  unMuteConversation: dispatchProps._onUnMuteConversation,
-})
+const usePhoneHeader = (props: Props): boolean =>
+  props.participants.length === 2 && props.participants.some(participant => participant.endsWith('@phone'))
```

Comment:
What about more than 2 participants?

Also: have we confirmed that `@phone` is the canonical way we are representing this?

Reply:
I'm not convinced it's worth the effort to make this header work with a combination of phone numbers and not-phone-numbers. I'll fix it to work with multiple phone numbers though.

I think it is `@phone`. @zapu ?

Reply:
is there a way we can not do stuff like this and make service return `serviceName` field or something?

where does this data come from?

Reply:
Yeah this isn't good, we can discuss tomorrow.